docs/sources/style-guide/voice-tone-guidelines/index.md

```diff
@@ -29,7 +29,9 @@ Write in a natural and conversational style with language that reflects a positi
 
 A casual tone is accepted in UI copy and supporting material compared to traditional business writing.
 
-As long as they allow for shorter sentences, you can use the words _and_, _but_, and _so_ in moderation.
+As long as they allow for shorter sentences, you can use the words _and_, _but_, and _so_ in moderation. 
+
+Use contractions like _isn't_ for _is not_ to help to convey an informal tone. But [be careful](https://www.grammarly.com/blog/contractions/).
```

Comment:
I would remove line 34 completely.  I was trained to avoid contractions in technical documentation for localization purposes.  Both IBM and Microsoft manuals of style recommend avoiding contractions because they can be confusing for non-native English speakers.

Reply:
About three or four of us chimed in on this topic in Slack recently, and the consensus was that our style was conversational and so contractions were preferred. Google style recommends contractions to give docs an informal tone, and we incorporate Google so that was the direction I took.

Reply:
Microsoft also recommends contractions, so their advice might have changed at some point. https://learn.microsoft.com/en-us/style-guide/word-choice/use-contractions

Reply:
Huh, I've been told for 20 years to avoid contractions.  If we're going for conversational then I guess we're using them.